lib/ukrand/dev.c

```diff
@@ -64,7 +38,7 @@ static int dev_random_write(struct device *dev __unused,
 			    struct uio *uio __unused,
 			    int flags __unused)
 {
-	return EPERM;
+	return -EPERM;
```

Comment:
Why does this function return an `EPERM` in the first place? And why did you change its sign? If there is a good reason for this please do so in a separate commit. This commit supposedly only migrates a library.

Reply:
> Why does this function return an EPERM in the first place?

Not sure, should we change this to `return 0` or some other error?

> And why did you change its sign? 

I think it was a checkpatch warning.

Reply:
Ok then this would mean I guess that the change is our of scope of this commit. You can choose to ignore it and undo the change, or create an additional commit giving a reason why you changed the sign, if you really see one.

Reply:
I reverted it at the time of the comment; I will resolve this.